libres/lib/enkf/obs_data.cpp

```diff
@@ -128,12 +131,11 @@ static void obs_block_free__(void *arg) {
     obs_block_free(obs_block);
 }
 
-void obs_block_deactivate(obs_block_type *obs_block, int iobs, bool verbose,
+void obs_block_deactivate(obs_block_type *obs_block, int iobs,
                           const char *msg) {
     if (obs_block->active_mode[iobs] == ACTIVE) {
-        if (verbose)
-            printf("Deactivating: %s(%d) : %s \n", obs_block->obs_key, iobs,
-                   msg);
+        logger->info("Deactivating: {}({}) : {} \n", obs_block->obs_key, iobs,
```

Comment:
This is perhaps a `WARNING`? It could lead to failure if all obs are deactivated.